app/src/sharedTest/java/com/example/android/architecture/blueprints/todoapp/statistics/StatisticsFragmentTest.kt

```diff
@@ -49,10 +45,11 @@ import org.junit.runner.RunWith
 @MediumTest
 @ExperimentalCoroutinesApi
 class StatisticsFragmentTest {
+
     private lateinit var repository: TasksRepository
 
-    // An Idling Resource that waits for Data Binding to have no pending bindings
-    private val dataBindingIdlingResource = DataBindingIdlingResource()
+    @get:Rule
```

Comment:
This file needs a complete cleanup. 

- Rename to StatisticsScreenTests
- Remove all test setup related to repository
- start a ComposeActivity and set its content manually, with the appropriate data

Reply:
All these Fragment tests need to be cleaned up at some point. Since they still require the Activity to interact with the TopBar, etc. can I refactor all of them when the migration is almost complete? It's going to be easier

Reply:
Nvm! I know what you mean :D fixed, thanks!